bika/lims/browser/bika_listing.py

```diff
@@ -588,13 +588,6 @@ def update(self):
         self.member = api.get_current_user()
         self.translate = self.context.translate
 
-        # Sanitize the request form
-        # XXX When does this happen?
-        if self.request.form:
-            for key, value in self.request.form.items():
-                if isinstance(value, list):
-                    self.request.form[key] = self.request.form[key][0]
-
```

Comment:
Ok, we need to keep that one in mind. I wanted to delete it as well during refactoring, but decided to add only the comment. Definitely a weird piece of code and I have no clue when this happens.